client/my-sites/stats/pages/realtime/style.scss

```diff
@@ -26,3 +26,28 @@
 	}
 	/* Style the line chart svg using the user admin color scheme */
 }
+
+@import "@automattic/typography/styles/variables";
+
+.stats-realtime-header {
+	margin-top: 32px;
+}
+
+.stats-realtime-header__title {
+	font-family: $brand-serif;
+	// stylelint-disable-next-line declaration-property-unit-allowed-list
+	font-size: 32px;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 40px;
+}
+
+.stats-realtime-header__description {
+	font-family: inherit;
+	// stylelint-disable-next-line declaration-property-unit-allowed-list
+	font-size: 14px;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 16px;
+	color: var(--color-text-subtle);
+}
```

Comment:
Would it help readability to nest these, eg. 

```
.stats-realtime-header {
    &__title {
    }
 ```

I know this is waiting on design feedback first, so definitely not a blocker,  but I think we should be able to use the font-size vars like  `font-size: $font-title-large;` etc. already?

Reply:
Thanks Anna. I was looking at the font styles and didn't find a match for the sizes I wanted. The goal was to match the size/style from the other tabs so I ended up hard-coding the values. There's a good chance I missed something though.

The existing headers are also somewhat inconsistent with spacing/padding/etc. It would be nice to revisit and consolidate these across the existing tabs.